azure-pipelines-v3.yml

```diff
@@ -1,3 +1,72 @@
+parameters:
```

Comment:
move this section to regression text job variables as it is local to that job only.

Reply:
No, this is also useful for warm up job.